lib/collection/property-base.js

```diff
@@ -169,6 +169,28 @@ _.assign(PropertyBase.prototype, /** @lends PropertyBase.prototype */ {
         return this && this.__parent && (this.__parent.__parent || this.__parent) || undefined;
     },
 
+    /**
+     * Returns the path of the item
+     *
+     * @returns {Array<string>}
+     */
+    getPath () {
```

Comment:
Why add this to `PropertyBase`? Do we want to add this to every type?

Reply:
Changed now